spec/system/news_article_spec.rb

```diff
@@ -126,5 +126,23 @@
 
       it_behaves_like "a news article page with a high resolution image"
     end
+
+    context "when visiting a page in history mode" do
+      let(:path) { "/government/news/final-care-act-guidance-published?graphql=true" }
+      # rubocop:disable RSpec/LetSetup
```

Comment:
Can you add something to the commit message about why this Cop has been disabled.

Reply:
(Particularly, can we double-check this isn't just a before block in disguise)

Reply:
Yes, agreed. I did wonder why a `let!` is being used in these tests. I've noticed that they've been littered about the tests with no explanation.

Reply:
Can you make sure that when the tests are added in [graphql-news-articles-meta-tags-etc-remainder](https://github.com/alphagov/frontend/compare/main...graphql-news-articles-meta-tags-etc-remainder) that you add something to the appropriate commit message about why `let!` is being used therefore why `RSpec/LetSetup` is being disabled. I had a quick look and at time of writing that hadn't been done.

Reply:
Yeah I just moved those onto another branch for reference. The intention isn't to open a PR with those changes at the moment given Keith's provocations on our broader approach to testing, but if we do, we'll be sure to address this